UI Session/UI Session4/css/style.css

```diff
@@ -0,0 +1,280 @@
+body {
+    margin: 0;
+    padding: 0;
+    font-family: 'Raleway', sans-serif;
+    background-image: url('../images/bodybg.png');
+}
+
+html {
+    scroll-behavior: smooth;
+}
+
+* {
+    box-sizing: border-box;
+}
+
+.heroSection {
+    background-image: url("../images/homebg.png");
+    background-size: 100% 100%;
+}
+
+header.hero-top {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+}
+
+li {
+    display: inline-block;
+}
+
+li, i {
+    margin-left: 10px;
+}
+
+.logo {
+    text-transform: uppercase;
+}
+
+
+.logo, li {
+    font-weight: bold;
+}
+
+a {
+    display: inline-block;
+    text-decoration: none;
+    color: #fff;
+    font-size: 13px;
+    transition: 400ms;
+}
+
+a:hover {
+    color: #1B1464;
+    transform: scale(1.4, 1.4);
+}
+
+h1 {
+    color: #fff;
+    font-size: 2em;
+}
+
+.hero-center {
+    padding: 100px;
+    display: flex;
+    align-items: top;
+    justify-content: space-around;
+}
+
+.phone {
+    height: 450px;
+    transition: 400ms;
+}
+
+.phone:hover {
+    transform: scale(1.1, 1.1);
+}
+
+span {
+    display: block;
+}
+
+button {
+    margin: 10px;
+    padding: 10px 30px;
+    border-radius: 3px;
+    border: none;
+    color: #fff;
+    transition: 400ms;
+}
+
+button:hover {
+    transform: scale(1.2, 1.2);
+    box-shadow: 0px 2px 10px #fff;
+}
+
+button.left {
+    background-color: #F97F51;
+}
+
+button.right {
+    background-color: #575fcf;
+}
+
+main {
+    margin: 80px auto;
+    width: 80%;
+}
+
+main header {
+    text-align: center;
+    padding-bottom: 100px;
+}
+
+main header h2 {
+    color: #686de0;
+}
+
+main p {
+    font-size: 14px;
+    color: #95a5a6;
+}
+
+.perfectSectionMiddle {
+    display: flex;
+    margin: auto;
+    justify-items: center;
+    justify-content: space-evenly;
+    text-align: justify;
+}
+
+.descriptionSection {
+    flex-basis: 25%;
+    display: flex;
+    align-items: top;
+}
+
+.descriptionSection h2 {
+    font-size: 20px;
+}
+
+.slider {
+    display: block;
+    margin: 50px auto;
+    width: 50%;
+}
+
+.feature {
+    /* padding: 0; */
+    margin: 0;
+    color: #7f8c8d;
+}
+
+.col2 {
+    padding-left: 10px;
+}
+
+.perfectSectionMiddle i {
+    font-size: 40px;
+}
+
+.topScreenshot, .bottomScreenshot, .middleScreenshot {
+    display: flex;
+    padding: 50px;
+    align-items: center;
+    justify-content: space-around;
+}
+
+.screenshotDescription {
```

Comment:
css class name should be like screen-shot-description. And if we use bootstrap 4 library we can give our classes in html like d-flex flex grow justify-content-around.